src/pytest_html/__init__.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from . import __version
+    from . import __version  # type: ignore
 
     __version__ = __version.version
```

Comment:
since `__version` is an optioanl import at the time typecheckers run, 
i usggest to typehint `__version__` ala 
```py
     __version__: str = __version.version 
```

this makes type checkers aware that `__version__` is supposed to be a sting and warn/fail if it is not